app/src/renderer/components/staking/UndelegationModal.vue

```diff
@@ -131,7 +133,7 @@ export default {
     },
     async submitForm(submitType, password) {
       await this.$store.dispatch(`submitUnbondingDelegation`, {
-        amount: -this.amount,
+        amount: -uatoms(this.amount),
```

Comment:
it's very weird to me that the REST API is expecting a negative number

Reply:
undelegating a negative number is a double negative 😖

Reply:
this is not from the API, just from our delegation or send module